Description: Enable PDF build from sources.
Author: Eugene Seliverstov <theirix@gmail.com>
Last-Update: 2013-06-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/docsrc/asn1c-usage.tex
+++ b/doc/docsrc/asn1c-usage.tex
@@ -6,9 +6,6 @@
 \usepackage[hmargin={1in,1in},vmargin={2in,2in}]{geometry}
 
 \defaultfontfeatures{Mapping=tex-text}
-\setmainfont{Myriad Pro}
-\setsansfont{Myriad Pro}
-\setmonofont{Consolas}
 
 \usepackage{fancyhdr}
 \pagestyle{fancy}
--- a/doc/docsrc/Makefile.am
+++ b/doc/docsrc/Makefile.am
@@ -2,6 +2,8 @@
 XELATEX_FLAGS = -halt-on-error -file-line-error
 XELATEX = xelatex
 
+all: regen
+
 regen:
 	$(XELATEX) $(XELATEX_FLAGS) -no-pdf asn1c-usage
 	$(XELATEX) $(XELATEX_FLAGS) -no-pdf asn1c-usage
